docs/en/week11/11-2.md

```diff
@@ -0,0 +1,382 @@
+
+* Cycles and self-loops are allowed
+* Multiple start and accept nodes are allowed
+* $\epsilon$ "nothing" transitions are allowed in WFSAs and WFSTs.
+
+## Operations:
+
+### Unions
+
+The union accepts a sequence if it is accepted by any of the input graphs. 
+
+<center>
+<img src="{{site.baseurl}}/images/week11/11-2/figure3.png" style="background-color:#DCDCDC;"/><br>
+<b>Figure 3:</b> Union Operation <br>
+<br>
+</center>
+
+
+### Kleene Closure
+
+Accepts any accepted sequence by input graph repeated 0 or more times.
+
+<center>
+<img src="{{site.baseurl}}/images/week11/11-2/figure4.png" style="background-color:#DCDCDC;"/><br>
+<b>Figure 4:</b> Kleene Closure Operation <br>
+<br>
+</center>
+
+
+### Intersect
+
+Similar to matrix multiplication or convolution. This operation is probably the most important in GTNs. 
+1. Any path that is accepted by both WFSAs is accepted by the product of the intersection.
+2. The score of the instersected path is the sum of the scores of the paths in the input graphs. 
+
+<center>
+<img src="{{site.baseurl}}/images/week11/11-2/figure5.png" style="background-color:#DCDCDC;"/><br>
+<b>Figure 5:</b> Intersection Operation <br>
+<br>
+</center>
+
+
+
+### Compose
+
+It is basically the same thing as intersection but for transducers instead of acceptors. 
+
+<center>
+<img src="{{site.baseurl}}/images/week11/11-2/figure6.png" style="background-color:#DCDCDC;"/><br>
+<b>Figure 6:</b> Compose Operation <br>
+<br>
+</center>
+
+The main thing about composition is that it let's map from different domains. For example, if we have a graph that maps from letters to words and another graph that maps from words to sentences. When we compose these two graphs we will map from letters to sentences. 
+
+### Forward Score
+
+Assumes the graph is **DAG** and an efficient dynamic programming algorithm.
+
+<center>
+<img src="{{site.baseurl}}/images/week11/11-2/figure7.png" style="background-color:#DCDCDC;"/><br>
+<b>Figure 7:</b> Forward Score <br>
+<br>
+</center>
+
+The graphs accepts three paths: 
+* **aca** with score = 1.1 + 1.4 + 2.1
+* **ba** with score = 3.2 + 2.1
+* **ca** with score 1.4 + 2.1
+
+ForwardScore(g) is the actual-softmax of the paths.
+
+
+## Sequence Criteria with WFSTs 
+
+### Target graph Y:
+
+<center>
+<img src="{{site.baseurl}}/images/week11/11-2/figure8.png" style="background-color:#DCDCDC;"/><br>
+<b>Figure 8:</b> Target Graph Y <br>
+<br>
+</center>
+
+
+### Emissions graph E:
+
+Think of it between two nodes we have logits.
+
+<center>
+<img src="{{site.baseurl}}/images/week11/11-2/figure9.png" style="background-color:#DCDCDC;"/><br>
+<b>Figure 9:</b> Emission Graph E <br>
+<br>
+</center>
+
+
+### Target contrained graph A by intersection(Y, E):
+
+<center>
+<img src="{{site.baseurl}}/images/week11/11-2/figure10.png" style="background-color:#DCDCDC;"/><br>
+<b>Figure 10:</b> Target Constrained Graph <br>
+<br>
+</center>
+
+
+### Loss
+
+$$
+loss = - (forwardScore(A) - forwardScore(E))
```

Comment:
"loss" is not a mathematical symbol. Variables have a single character. Use `\ell` with correct arguments.
Also, use `\text{}` for the function names.